docs/teams/emr-eks-team.md

```diff
@@ -0,0 +1,85 @@
+# EMR on EKS Team
+
+The `EMR on EKS Team` extends the `ApplicationTeam` and allow the EMR on EKS team to manage the namespace where the virtual cluster is deployed. This team **MUST** be used by in conjuction with EMR on EKS AddOn.
+
+The EMR on EKS Team allow you to create a [Virtual Cluster](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/virtual-cluster.html) and job Execution Roles that are used by the job to access data in Amazon S3, AWS Glue Data Catalog or any other AWS resources that you need to interact with. The job execution role are scoped with IRSA to be only assumed by pods deployed by EMR on EKS in the namespace of the virtual cluster. You can learn more about the condition applied [here](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/iam-execution-role.html). 
+
+
+## Usage
+
+```typescript
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.EmrEksAddOn();
+
+//The policy to be attached to the EMR on EKS execution role 
+const executionRolePolicyStatement: PolicyStatement [] = [
+            new PolicyStatement({
+              resources: ['*'],
+              actions: ['s3:*'],
+            }),
+            new PolicyStatement({
+              resources: ['*'],   
+              actions: ['glue:*'],
+            }),
+            new PolicyStatement({
+              resources: ['*'],
+              actions: [
+                'logs:*',
+              ],
+            }),
+          ];
+      
+      const dataTeam: EmrEksTeamProps = {
+              name:'dataTeam',
+              virtualClusterName: 'batchJob',
+              virtualClusterNamespace: 'batchjob',
+              createNamespace: true,
+              users: [
+                new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user1`),
+                new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:role/role1`)
```

Comment:
Can't use role under the users as a principal, we have a separate `userRoleArn` attribute for that. Please let me know if it was copied over from our own docs, I will fix it.

Reply:
I did not copy, I was not aware of this limitation. Do you think we can have `userRoleArn` as a list also? Fixed this.